src/ngraph/op/reduce_logical_and.hpp

```diff
@@ -0,0 +1,53 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/op/util/arithmetic_reductions_keep_dims.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v1
+        {
+            /// \brief Performs a reduction using "logical and"
+            ///
+            /// The reduction is performed over slices of the first input. The slices shape depends
+            /// on the values passed to the second input - the axes.
+            class ReduceLogicalAnd : public util::ArithmeticReductionKeepDims
```

Comment:
I don't see changes related to constant folding passes for this new ops, so please add it. Also according to "nGraph/DLDT common operation set" table you have implementation for v0 ops.

Reply:
Done, please have a look.